Common/src/CConfig.cpp

```diff
@@ -3316,6 +3333,35 @@ void CConfig::SetPostprocessing(SU2_COMPONENT val_software, unsigned short val_i
     VolumeOutputFiles[2] = OUTPUT_TYPE::SURFACE_PARAVIEW_XML;
   }
 
+  /*--- Set the default output frequencies ---*/
+  if (!OptionIsSet("OUTPUT_WRT_FREQ")){
+    nVolumeOutputFrequencies = nVolumeOutputFiles;
+    VolumeOutputFrequencies = new unsigned long [nVolumeOutputFrequencies];
+
+    /*---  Using default frequency of 250 for all files when steady, and 1 for unsteady. ---*/
+    for (auto iVolumeFreq = 0; iVolumeFreq < nVolumeOutputFrequencies; iVolumeFreq++){
+      if (Time_Domain) 
+        VolumeOutputFrequencies[iVolumeFreq] = 1; 
+      else
+        VolumeOutputFrequencies[iVolumeFreq] = 250; 
+    }
+  } else {
+    /*--- check how many frequencies. If 1 then use it for all output ---*/
+    if (nVolumeOutputFrequencies == 1) {
+      unsigned short outputWriteFreq = VolumeOutputFrequencies[0];
+      /*--- we recreate the OutputFrequencies ---*/
+      nVolumeOutputFrequencies = nVolumeOutputFiles;
+      delete [] VolumeOutputFrequencies;
+      VolumeOutputFrequencies = new unsigned long [nVolumeOutputFrequencies];
+
+      for (auto iVolumeFreq = 0; iVolumeFreq < nVolumeOutputFrequencies; iVolumeFreq++){
+        VolumeOutputFrequencies[iVolumeFreq] = outputWriteFreq; 
+      }
+    } else if (nVolumeOutputFrequencies != nVolumeOutputFiles) {
```

Comment:
It would be convenient to repeat the last frequency specified, that would allow you specify the restart first and then have the same frequency for all the paraview files